chainer/training/extensions/evaluator.py

```diff
@@ -202,6 +202,11 @@ def evaluate(self):
             iterator.reset()
             it = iterator
         else:
+            warnings.warn('This iterator does not have the reset method.' +
```

Comment:
Concatenation does not insert spaces between.
Also it's better to avoid `+`, as multiline strings are automatically concatenated in Python.

I.e. Please write as follows.
(I like breaking the line after `(`, but it's a matter of preference.)

```
             warnings.warn('This iterator does not have the reset method. '
                           'Evaluator copies the iterator instead of '
                           'resetting. This behavior is deprecated. '
                           'Please implement the reset method.',
                           DeprecationWarning)
```

Reply:
Done! Thanks.